src/test/java/uk/gov/companieshouse/officerssearch/subdelta/search/SearchApiClientTest.java

```diff
@@ -54,6 +56,7 @@ class SearchApiClientTest {
     @BeforeEach
     void setup() {
         when(clientSupplier.get()).thenReturn(apiClient);
+        when(apiClient.getHttpClient()).thenReturn(httpClient);
```

Comment:
There are other lines of stubbing in this class which could be moved to the setup() method also. For example line 60, and the other tests that line appears in.

Reply:
Same issue here, Mockito complains about unnecessary stubbing